project-keeper/src/main/java/com/exasol/projectkeeper/ProjectKeeper.java

```diff
@@ -137,6 +138,8 @@ private ValidationPhase phase2AnalyzeSources(final ValidationPhase.Provision pro
         final String projectName = getProjectName(analyzedSources);
         final var brokenLinkReplacer = new BrokenLinkReplacer(this.config.getLinkReplacements());
         final String projectVersion = new ProjectVersionDetector().detectVersion(this.config, analyzedSources);
+        // [impl->dsn~verify-modes.output-parameters~1]
+        GitHubWorkflowOutputPublisher.create(this.config, projectDir, projectVersion, analyzedSources).publish();
```

Comment:
I think this is exactly the topic we discussed, right? 
Now, when reading the method name "analyze" I stumble over `GitHubWorkflowOutputPublisher.publish()`, because publishing something sounds strange in the context of analyzing something else. 

Could we either add a comment or encapsulate the publish call into another method?

Maybe it would help to rename the class `GitHubWorkflowOutputPublisher` to `GitHubWorkflowOutput` and the method `publish()` to `provide()`?

Reply:
Also, the package name sound a little bit too abbreviated and generalized to me. 

Proposal: 
* Package name `github` 
* optional subpackage: `workflow`
* Interface name `WorkflowOutput`
* classes `GitHubWorkflowOutput`, `FileOutput`, `NullOutput`